templates/vue/src/components/Lightbox/media/VideoMedia/H5PVideoMedia/H5PIframe.vue

```diff
@@ -93,6 +99,39 @@ export default {
     document.removeEventListener("mozfullscreenchange", this.setFrameDimensions)
   },
   methods: {
+    startTimeupdateHandler() {
+      this.stopTimeupdateHandler()
+      this.interval = setInterval(() => this.updateVideoProgress(), 200)
+    },
+    stopTimeupdateHandler() {
+      if (this.interval) {
+        clearInterval(this.interval)
+      }
+    },
+    updateVideoProgress() {
+      const video = this.getInstance()
+
+      /**
+       * An H5P video is just one of the many types H5Ps can be, so we only want to
+       * trigger this if the H5P is in fact a video.
+       */
+      if (video) {
```

Comment:
Is this true anymore? Should we still have this check in place if we know it's a video?

Reply:
This one seems to be the only check that is necessary throughout the component. When clicking an interactive content on the video and the closing it the interval keeps running. I added an else clauses to stop the interval if the video does not exists. 

To sum it up, the check is necessary for something else and this comment is inaccurate.